baystation12.dme

```diff
@@ -3488,6 +3488,21 @@
 #include "code\modules\urist\modules\weather\weather_helpers.dm"
 #include "code\modules\urist\modules\weather\weather_objects.dm"
 #include "code\modules\urist\modules\weather\weather_turf.dm"
+#include "code\modules\urist\modules\wfc\_wfc_defines.dm"
+#include "code\modules\urist\modules\wfc\fractal_coords.dm"
+#include "code\modules\urist\modules\wfc\core\calllib.dm"
+#include "code\modules\urist\modules\wfc\core\maptricks.dm"
+#include "code\modules\urist\modules\wfc\core\prefabs.dm"

```

Comment:
Not sure what's going on here. But I think your includes have got very mixed up?
Many of these are included in `code/modules/urist/modules/wfc/_module.dm` already, however that file isn't ever included anywhere? Some of these includes are behind `ifdef`s too. Are these all supposed to be in the dme, rather than a single module file which then further includes?

Might need a pass?

Reply:
I left a comment in the module file and the commit message 😅 

TL;DR - DM was acting weird with call()s on code included from the submodule, and this had been hanging around for a long time now so I'd rather get it in and fix the includes separately.

The current `_module.dm` is NOT included to avoid double-including and the tick-included featureset corresponds to the correct SS13 set of includes.